test/Makefile

```diff
@@ -16,9 +16,9 @@ ifdef CMD_EXE
   RMDIR = rmdir /s /q $(subst /,\,$1)
 else
   EXE :=
-  DEL = $(RM) $1
+  DEL = $(RM) -f $1
```

Comment:
On Friday 10 July 2015, 02:04:02 Oliver Schmidt <notifications@github.com> 
wrote:
> > @@ -16,9 +16,9 @@ ifdef CMD_EXE
> > 
> >    RMDIR = rmdir /s /q $(subst /,\,$1)
> >  
> >  else
> >  
> >    EXE :=
> > 
> > -  DEL = $(RM) $1
> > +  DEL = $(RM) -f $1
> 
> According to the GNU make doc the default value for `RM` is `rm -f`. That's
> the reason why I introduced `$(RM)` in the first place. Are you sure that
> the `-f`is actually necessary?

oh! you are correct, it is not... i also removed the (hopefully now 
unnecessary) "-" prefix in the clean rule ... someone might want to test if it 
still works on windows now :)

-- 

http://www.hitmen-console.org    http://magicdisk.untergrund.net
http://www.pokefinder.org        http://ar.pokefinder.org

Some people say that I must be a horrible person, but that's not true. I have 
the heart of a small boy --- in a jar on my desk 
<Stephen King>